src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -16,7 +16,7 @@
 
     @Argument(fullName = "maxFragmentLength",
             shortName = "maxFragmentLength",
-            doc = "Keep only read pairs with fragment length at most equal to the given value",
+            doc = "Keep only read pairs with fragment length at most equal to the given value.",
```

Comment:
This "long" explanation belongs in the filter description (L11). The argument doc string should pertain strictly to the argument being described. E.g. here you would say something like "Maximum length of fragment acceptable". 

By the way, the filter description should clarify what "fragment" refers to here. Is it a fragment of read? Or is it actually referring to the sequencing insert length? In which case, why isn't this called the InsertLengthReadFilter?